src/tests/controllers/ArticleController.test.js

```diff
@@ -177,6 +174,44 @@ describe('Article', () => {
         done();
       });
   });
+
+  it('Should update article cover', async () => {
+    const myArticle = await db.Article.findOne({ where: { userId: createdUser.id } });
+    chai
+      .request(app)
+      .post(`/api/v1/articles/${myArticle.dataValues.slug}/cover`)
+      .set('access-token', accessToken)
+      .send({ coverUrl: 'placeholder.png' })
+      .end((err, res) => {
+        expect(res).to.have.status(status.OK);
+        res.body.should.be.an('object');
+        expect(res.body.coverUrl).to.equal('CoverUrl has been updated');
+      });
+  });
+  it('Should not update article cover if image not provided ', async () => {
+    chai
+      .request(app)
+      .post('/api/v1/articles/rosie-make-it-easy-1dh6jv9cn4sz/cover')
+      .set('access-token', accessToken)
+      .send({ coverUrl: 'placeholder.png' })
+      .end((err, res) => {
+        expect(res).to.have.status(status.BAD_REQUEST);
+        res.body.should.be.an('object');
+        expect(res.body.errors.coverUrl).to.equal('coverUrl not updated, try again');
+      });
+  });
+  it('Should get gallery', async () => {
+    chai
+      .request(app)
+      .get('/api/v1/gallery')
+      .set('access-token', accessToken)
+      .end((err, res) => {
+        console.log('rw', res.body);
```

Comment:
remove console log and commented out code.

Reply:
@e-liyai solved